docusaurus/docs/themes/transportation/travel-modes.mdx

```diff
@@ -1 +1,271 @@
+is relatively small and contains only travel modes which we some
+confidence will be generally-applicable both across jurisdictions and
+over time. We expect the list to change in future schema updates, to be
+larger once we reach a table `v1.0.0` schema, and to continue to expand
+as the world changes and new information arises.
+
+Our criteria for recognizing a proposed new travel mode are:
+
+1. The proposed new travel mode should represent a *generally-accepted*
+   concept meaning that the concept exists in many places with roughly
+   the same definition.
+2. The proposed new travel mode should be referred to with well-known
+   sign iconography or sign text in those jurisdictions where the
+   concept exists; and this iconography or sign text is preferably
+   somewhat consistent across many jurisdictions and over time.
+3. It must be possible to give a relatively precise *definition* to the
+   proposed new travel mode and explain how it *differs* from, or
+   *intersects* with, other similar travel modes which are already
+   recognized in the Overture schema.
+4. The proposed new travel mode should preferably not be trivially
+   expressible using another existing
+   [Scoping Property](scoping-rules#scoped-values-and-scoping-properties).
+   (*An example of such a trivially-expressible concept might be a
+   "three-axle vehicle" which is more appropriately phrased as
+   `vehicle: { axleCount: { isEqualTo: 3 } }`.*)
+5. The proposed new travel mode must be supported by existing data in at
+   least one of Overture's upstream data sources.
+
+There is a risk that the above-described approach will be too slow to
+adapt to our users' needs in a vast, ever-changing world. One possible
+avenue to mitigate this risk is supporting
+[custom extensions](#should-we-support-extension-travel-modes) to the
+built-in list of recognized travel modes.
+
+## Travel mode schema
+
+### Scoping to travel modes
+
+Travel modes can used to change the scope of scoped and rule-based
+properties within the schema. For example, they can affect the scope of
+access restrictions, turn restrictions, or speed limits on [road](roads)
+segments. Since travel modes are expressed via scoping properties, it
+is recommended to read the section on
+[Scoped and Rule-Based Properties](scoping-rules) before proceeding
+further.
+
+#### Travel mode scoping: `mode` and `modeNot`
+
+Two scoping properties control whether a given scoped property applies
+to a given travel mode: `mode` and `modeNot`.
+
+If either `mode` or `modeNot` property is specified, the property value
+must be a non-empty, array of travel modes, and is interpreted as a set.
+Values must be unique, but order is not important.
+
+The `mode` and `modeNot` properties are not mutually exclusive: in
+places where scoping by travel mode is allowed, either `mode` or
+`modeNot`, or both, may be provided.
+
+1. **Neither**. When *neither* `mode` nor `modeNot` is provided, the
```

Comment:
Wouldn't it be more accurate to say when neither `mode` or `modeNot` is provided the parent property's scope includes all _default_ or _implied_ travel modes for that segments 'class'?

Reply:
Yes, *much* more accurate. Thanks for catching that. Fixing.